packages/client/src/utils/helpers.ts

```diff
@@ -1,3 +1,10 @@
-export const getFullUrlToResource = (url: string, resourceBaseUrl: string) => {
+export const getFullUrlToResource = (
+  url: string | null,
+  resourceBaseUrl: string
+) => {
```

Comment:
```suggestion
  base: string
): string => {
```
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#syntax